testing/py3-fastavro/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: Gennady Feldman <gena01@gmail.com>
+# Contributor: Gennady Feldman <gena01@gmail.com>
+pkgname=py3-fastavro
+_pkgname=fastavro
+pkgver=0.21.24
+pkgrel=0
+pkgdesc="Fast Avro for Python"
+url="https://github.com/fastavro/fastavro"
+arch="all"
+license="MIT"
+depends="py3-six"
+makedepends="py3-setuptools python3-dev cython3"
+checkdepends="py3-pytest py3-numpy"
+source="$pkgname-$pkgver.tar.gz::https://github.com/fastavro/fastavro/archive/$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	FASTAVRO_USE_CYTHON=1 python3 setup.py build
+}
+
+check() {
+	FASTAVRO_USE_CYTHON=1 python3 setup.py build_ext --inplace
+	PYTHONPATH=${PWD} python3 -m pytest -v tests
```

Comment:
can't PYTHONPATH be set to the path of where the build() installed it

Reply:
This is based on how the repo tests the code itself: https://github.com/fastavro/fastavro/blob/master/run-tests.sh#L21